components/sponsored_benefits/app/controllers/sponsored_benefits/organizations/plan_design_proposals/plan_reviews_controller.rb

```diff
@@ -50,7 +50,12 @@ def show
       def estimated_employee_cost_details
         @benefit_group = benefit_group
         @plan_cost_service = SponsoredBenefits::Services::PlanCostService.new({benefit_group: benefit_group})
-        @reference_plan = params[:kind] == 'dental' ? @benefit_group.dental_reference_plan : @plan_cost_service.reference_plan
+        if params[:kind] == 'dental'
+          @reference_plan = @benefit_group.dental_reference_plan
+          @kind = 'dental'
+        else
+          @reference_plan = @plan_cost_service.reference_plan
+        end
```

Comment:
just to be consistent you can alway set this `@kind` instance variable. something like below -

```
@kind = params[:kind]
@reference_plan = @kind == 'dental' ? @benefit_group.dental_reference_plan : @plan_cost_service.reference_plan
```